frontend/src/widgets/GoalStatusGraph.js

```diff
@@ -221,4 +221,4 @@ GoalStatusChart.defaultProps = {
   },
 };
 
-export default withWidgetData(GoalStatusChart, 'goalStatusGraph');
+export default withWidgetData(GoalStatusChart, 'goalStatusByGoalName');
```

Comment:
Assuming we wanted to change this name on an existing widget (ID).

Reply:
There are two goal status widgets now, I changed the name on the backend to make it more descriptive